src/k8s-extension/azext_k8s_extension/partner_extensions/AzureMLKubernetes.py

```diff
@@ -189,12 +191,12 @@ def __validate_config(self, configuration_settings, configuration_protected_sett
 
     def __validate_scoring_fe_settings(self, configuration_settings, configuration_protected_settings):
         experimentalCluster = _get_value_from_config_protected_config(
-            'inferenceLoadBalancerHA', configuration_settings, configuration_protected_settings)
-        experimentalCluster = str(experimentalCluster).lower() == 'true'
-        if experimentalCluster:
-            configuration_settings['clusterPurpose'] = 'DevTest'
-        else:
+            self.inferenceLoadBalancerHA, configuration_settings, configuration_protected_settings)
+        notExperimentalCluster = str(experimentalCluster).lower() == 'true'
```

Comment:
Something is wrong here, either the variable name or what you are checking, but this statement doesn't quite make sense

Reply:
Lia, is this a typo? It is odd to me too.

Reply:
ahhh yes sorry this is a typo. For got to rename experimentalCluster -> notExperimentalCluster in one extra place

Reply:
fixed to a better variable name altogether - isProdCluster